interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -388,6 +414,23 @@ function ViewModel() {
             self.isRestarting(1)
         }).always(self.setNextUpdate);
 
+        // this is a test of getting the entire queue
```

Comment:
Sorry I think I caused confusion. The behavior of the API is definitely as expected, because if a user drags (or uses the numbered drop-downs in smpl skin) he wants the job to move to wherever he just chose. Not for it to then jump down again if the priority doesn't match, so it has to change priority like this. 
So my comment from a while ago that the Javascript should take care of the priority stuff is wrong, the API will take care of it. No need to be smart, just move all the way to the top/bottom!